### PR TITLE
Change some language about autoconfig to be clearer

### PR DESCRIPTION
**Pull request by winny-**
Very minor nit. qutebrowser is my main interface into the awkward mouse-driven modern WWW. Thank you all for this wonderful program.

---
This change is [Reviewable](https://reviewable.io/reviews/qutebrowser/qutebrowser/4431)

### CODE REVIEW

File: doc/help/configuring.asciidoc [hunk 1]
```diff
@@ -19,10 +19,10 @@ hand, you can simply use those - see
 <<autoconfig,"Configuring qutebrowser via the user interface">> for details.
 
 For more advanced configuration, you can write a `config.py` file - see
-<<configpy,"Configuring qutebrowser via config.py">>. As soon as a `config.py`
+<<configpy,"Configuring qutebrowser via config.py">>. When `config.py`
```
Comment (The-Compiler) [inline doc/help/configuring.asciidoc]:
```suggestion
<<configpy,"Configuring qutebrowser via config.py">>. When a `config.py`
```

File: doc/help/configuring.asciidoc [hunk 2]
```diff
@@ -229,18 +229,18 @@ Loading `autoconfig.yml`
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
 All customization done via the UI (`:set`, `:bind` and `:unbind`) is
-stored in the `autoconfig.yml` file, which is not loaded automatically as soon
-as a `config.py` exists. If you want those settings to be loaded, you'll need to
-explicitly load the `autoconfig.yml` file in your `config.py` by doing:
+stored in file `autoconfig.yml`. When file `config.py` exists, `autoconfig.yml`
```
Comment (The-Compiler) [inline doc/help/configuring.asciidoc]:
```suggestion
stored in the `autoconfig.yml` file. When a `config.py` file exists, `autoconfig.yml`
```

File: doc/help/configuring.asciidoc (hunk 1 above)
Comment (winny-) [reply doc/help/configuring.asciidoc]: Wow English. Thanks for the review!

**Comment by The-Compiler** [comment]
Thanks!

**Pull request merged**